WtWSMS_EBS/history/characters/roman.txt

```diff
@@ -4161,8 +4190,8 @@
 	}
 }
 1300020 = {
-	name="Helpidius" #PLRE vol.2 Helpidius 1
-	#?consularis Campaniae a. 396, ?proconsul Africae 402
+	name="Helpidius" PLRE vol.2 Helpidius 1 
```

Comment:
Comments don't work without `#`, and there's no need for spaces in the end of the line.